insights-notification-schemas-java/src/main/resources/schemas/ActionDetails.json

```diff
@@ -0,0 +1,50 @@
+{
+  "$comment": "This schema defines some additional information for an Action that might be useful",
+  "$schema": "https://json-schema.org/draft-07/schema#",
+  "additionalProperties": false,
+  "description": "Describes additional information fields for some of the fields of the Action",
+  "properties": {
+    "action": {
+      "$ref": "Action.json"
```

Comment:
I was thinking something like:

`Action-out.json`
```
{
  "$comment": "This schema defines the data sent to integrations (or similar)", 
  "allOf": [{ "$ref": "./Action.json" }],
  "properties": {
       "source": "eveything you had in source in the previous code"
   }
}
```

Reply:
I am sorry... hopefully the version I Just pushed makes more sense now :smile: